_posts/2018-08-26-eager-dcgan/eager-dcgan.Rmd

```diff
@@ -324,33 +323,33 @@ Note that all model calls happen inside `tf$GradientTape` contexts. This is so t
 Obtain the gradients of the losses to the respective models' variables (`tape$gradient`) and have the optimizers apply them to the models' weights (`optimizer$apply_gradients`):
 
 ```{r}
-      gradients_of_generator <-
-        gen_tape$gradient(gen_loss, generator$variables)
-      gradients_of_discriminator <-
-        disc_tape$gradient(disc_loss, discriminator$variables)
+gradients_of_generator <-
+gen_tape$gradient(gen_loss, generator$variables)
+gradients_of_discriminator <-
+disc_tape$gradient(disc_loss, discriminator$variables)
       
-      generator_optimizer$apply_gradients(purrr::transpose(
-        list(gradients_of_generator, generator$variables)
-      ))
-      discriminator_optimizer$apply_gradients(purrr::transpose(
-        list(gradients_of_discriminator, discriminator$variables)
-      ))
+generator_optimizer$apply_gradients(purrr::transpose(
+  list(gradients_of_generator, generator$variables)
+))
+discriminator_optimizer$apply_gradients(purrr::transpose(
+  list(gradients_of_discriminator, discriminator$variables)
+))
       
-      total_loss_gen <- total_loss_gen + gen_loss
-      total_loss_disc <- total_loss_disc + disc_loss
+total_loss_gen <- total_loss_gen + gen_loss
+total_loss_disc <- total_loss_disc + disc_loss
 })
```

Comment:
I think having an end block with no beginning block is confusing. I'd just drop this entirely.